src/azure-cli/azure/cli/command_modules/cosmosdb/_help.py

```diff
@@ -204,3 +204,437 @@
+type: command
+short-summary: Get the throughput of the SQL database.
+"""
+
+helps['cosmosdb sql database throughput update'] = """
+type: command
+short-summary: Update the throughput of the SQL database.
+"""
+
+# SQL container throughput
+helps['cosmosdb sql container throughput'] = """
+type: group
+short-summary: Manage Azure Cosmos DB SQL container throughput.
+"""
+
+helps['cosmosdb sql container throughput show'] = """
+type: command
+short-summary: Get the throughput of the SQL container.
+"""
+
+helps['cosmosdb sql container throughput update'] = """
+type: command
+short-summary: Update the throughput of the SQL container.
+"""
+
+helps['cosmosdb mongodb'] = """
+type: group
+short-summary: Manage Azure Cosmos DB MongoDB resources.
+"""
+
+# MongoDB database
+helps['cosmosdb mongodb database'] = """
+type: group
+short-summary: Manage Azure Cosmos DB MongoDB databases.
+"""
+
+helps['cosmosdb mongodb database create'] = """
+type: command
+short-summary: Create an MongoDB database under an Azure Cosmos DB database account.
+"""
+
+helps['cosmosdb mongodb database list'] = """
+type: command
+short-summary: List the MongoDB databases under an Azure Cosmos DB database account.
+"""
+
+helps['cosmosdb mongodb database show'] = """
+type: command
+short-summary: Show the details/properties of a MongoDB database under an Azure Cosmos DB database account.
+"""
+
+helps['cosmosdb mongodb database delete'] = """
+type: command
+short-summary: Delete the MongoDB database under an Azure Cosmos DB database account.
+"""
+
+# MongoDB collection
+helps['cosmosdb mongodb collection'] = """
+type: group
+short-summary: Manage Azure Cosmos DB MongoDB collections.
+"""
+
+helps['cosmosdb mongodb collection create'] = """
+type: command
+short-summary: Create an MongoDB collection under an Azure Cosmos DB MongoDB database.
+examples:
+  - name: Create an Azure Cosmos DB MongoDB collection.
+    text: az cosmosdb mongodb collection create -g MyResourceGroup -a MyAccount -d MyDatabase -n MyCollection --shard "ShardingKey" --idx @indexes-file.json --throughput "500"
+    crafted: true
+"""
+
+helps['cosmosdb mongodb collection update'] = """
+type: command
+short-summary: Update an MongoDB collection under an Azure Cosmos DB MongoDB database.
+"""
+
+helps['cosmosdb mongodb collection list'] = """
+type: command
+short-summary: List the MongoDB collections under an Azure Cosmos DB MongoDB database.
+"""
+
+helps['cosmosdb mongodb collection show'] = """
+type: command
+short-summary: Show the details/properties of a MongoDB collection under an Azure Cosmos DB MongoDB database.
+"""
+
+helps['cosmosdb mongodb collection delete'] = """
+type: command
+short-summary: Delete the MongoDB collection under an Azure Cosmos DB MongoDB database.
+"""
+
+# MongoDB database throughput
+helps['cosmosdb mongodb database throughput'] = """
+type: group
+short-summary: Manage Azure Cosmos DB MongoDB database throughput.
```

Comment:
Consider something like "Manage throughput of MongoDB database under a Azure Cosmos DB account."